### PR TITLE
Adding libaio installation to Dockerfiles

### PR DESCRIPTION
**Pull request by TalZaccai**
Installing libaio in Linux images

Fix https://github.com/microsoft/garnet/issues/547

### CODE REVIEW

File: Dockerfile.chiseled [hunk 1]
```diff
@@ -29,6 +29,7 @@ RUN dotnet publish -a $TARGETARCH -c Release -o /app --no-restore --self-contain
 
 # Final stage/image
 FROM mcr.microsoft.com/dotnet/runtime:8.0-jammy-chiseled AS runtime
+
```
Comment (badrishc) [inline Dockerfile.chiseled:32]: why is chiseled not installing the required libraries?

File: Dockerfile.ubuntu [hunk 2]
```diff
@@ -29,6 +29,10 @@ RUN dotnet publish -a $TARGETARCH -c Release -o /app --no-restore --self-contain
 
 # Final stage/image
 FROM mcr.microsoft.com/dotnet/runtime:8.0-jammy AS runtime
+
+# Install libaio
+RUN apt-get update && apt-get install -y libaio1 && rm -rf /var/lib/apt/lists/*
```
Comment (badrishc) [inline Dockerfile.ubuntu:34]: why is this libaio1 instead of libaio?

Comment (TalZaccai) [reply Dockerfile.ubuntu:34]: That's just the package name in ubuntu

File: Dockerfile.chiseled (hunk 1 above)
Comment (TalZaccai) [reply Dockerfile.chiseled:32]: Following our convo, not needed in chiseled

**Review by badrishc — APPROVED** [review]

**Pull request merged**